includes/REST_API/Stories_Media_Controller.php

```diff
@@ -274,6 +274,71 @@ public function prepare_item_for_response( $post, $request ) {
 		return apply_filters( 'web_stories_rest_prepare_attachment', $response, $post, $request );
 	}
 
+	/**
+	 * Prepares links for the request.
+	 *
+	 * @since 1.12.0
+	 *
+	 * @param WP_Post $post Post object.
+	 *
+	 * @return array Links for the given post.
+	 */
+	protected function prepare_links( $post ): array {
+		$links = parent::prepare_links( $post );
+		$links = $this->add_taxonomy_links( $links, $post );
+
+		return $links;
+	}
+
+	/**
+	 * Adds a REST API links for the taxonomies.
+	 *
+	 * @since 1.12.0
+	 *
+	 * @param array   $links Links for the given post.
+	 * @param WP_Post $post Post object.
+	 *
+	 * @return array Modified list of links.
+	 */
+	private function add_taxonomy_links( array $links, WP_Post $post ): array {
```

Comment:
A test for this would be good, like we have for `Stories_Controller`

Reply:
Copied the test over 👍